ihatemoney/models.py

```diff
@@ -94,6 +94,17 @@ def members_stats(self):
             for member in self.active_members
         ]
 
+    @property
+    def monthly_stats(self):
+        """Compute expenses by month
+        """
+        monthly = defaultdict(lambda: defaultdict(int))
+
+        for member in self.active_members:
+            for bill in self.get_member_bills(member.id).all():
```

Comment:
Is it possible to rewrite this so that we do a unique SQL request rather than one per members?

Reply:
Probably; I will change it.

Reply:
I've changed this - as it turned out, a query for bills by active members was simply wrong for the historical statistics. I now just use `get_bills()`.